admin_interface/templates/admin/base_site.html

```diff
@@ -49,21 +51,30 @@
 
         {% if theme.logo_max_height > 0 %}
             --admin-interface-log-max-height: {{ theme.logo_max_height }}px;
+        {% else %}
+            --admin-interface-log-max-height: unset;
         {% endif %}
 
         {% if theme.related_modal_rounded_corners %}
             --admin-interface-related-modal-border-radius: 4px;
+        {% else %}
+            --admin-interface-related-modal-border-radius: 0px;
         {% endif %}
 
         {% if theme.css_module_rounded_corners %}
             --admin-interface-module-border-radius: 4px;
-            --admin-interface-border-radius-0px: 0px;
             --admin-interface-jsoneditor-border-radius: var(--admin-interface-module-border-radius);
             --admin-interface-jsoneditor-overflow: hidden;
+        {% else %}
+            --admin-interface-module-border-radius: 0px;
+            --admin-interface-jsoneditor-border-radius: var(--admin-interface-module-border-radius);
```

Comment:
Am I wrong or here `--admin-interface-jsoneditor-border-radius` should be `0px`?